app/portainer/home/EnvironmentList/KubeconfigButton.css

```diff
@@ -0,0 +1,18 @@
+.center-dialog {
```

Comment:
why don't you use css modules? these classes are global, with modules it will be scoped to the component

Reply:
Done, css modules has defined now